src/common/serac_types.hpp

```diff
@@ -47,6 +47,13 @@ enum class Preconditioner
   BoomerAMG
 };
 
+enum class CouplingScheme
+{
+  OperatorSplit,
+  FixedPoint,
+  FullyCoupled
```

Comment:
I'm curious to see what you have in mind for these coupling schemes!

Reply:
Ha! Me too! It's incredibly vague and notional in my head at this point...